### PR TITLE
Show that "port" attribute of "bind" tag also supports port ranges

### PR DESCRIPTION
**Pull request by Whissi**
The "port" attribute allows for port ranges, too.

### CODE REVIEW

File: docs/3/configuration/_bind.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ attributes:
   description: |-
     *TCP/IP listeners only!* If defined then the IP address to bind to instead of listening on all available interfaces.
 - name: port
-  type: Number
+  type: No. Range
   required: false
   default: null
   description: |-
```
Comment (genius3000) [inline docs/3/configuration/_bind.yml:15]: Should update the description wording to "port or range of ports".

**Review by genius3000 — CHANGES_REQUESTED** [review]
Nice catch! The only comment I have is to update the description as well. I left a comment on the v3 file; duplicate the changes to v4.

**Comment by SadieCat** [comment]
I think it'd be helpful for readers if you left the original example as-is and then added your new example which is binding on multiple ports below that so its clear that they don't necessarily *need* to bind on multiple ports.

**Comment by SadieCat** [comment]
Merged, thanks for your contribution! 👍

**Pull request merged**